CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Implement del method to release C++ client and server. Important for ray actors, because they create numerous clients during training.
 
+- If sparse feature values present on multiple servers, then only one will be returned with source picked randomly.
```

Comment:
Isn't it a user error if they load feature values to multiple servers with the same index?

Reply:
yes, but it is hard for them to fix it, so we can do our best to resolve this issue.